### PR TITLE
Ch setup database 163723794

### PR DESCRIPTION
**Pull request by ibkadeeko**
#### What does this PR do?
Create Local Database and persist data
#### Description of Task to be completed?
- Install and setup database
- Connect application to local database
- Persist current data structure parties and offices
#### What are the relevant pivotal tracker stories?
#163723794

### CODE REVIEW

File: server/controllers/offices.js [hunk 1]
```diff
@@ -1,58 +1,69 @@
...
 
-// Create office
-class offices {
-  static create(req, res) {
-    const alreadyExists = officesDb
-      .find(office => office.name.toLowerCase() === req.body.name.toLowerCase());
+const pool = new Pool({
+  user: 'ibukun', // this is the db user credential
+  database: 'politico',
+  password: 'password',
```
Comment (tutugodfrey) [inline server/controllers/offices.js:6]: why are you exposing your db password?

File: server/controllers/parties.js [hunk 2]
```diff
@@ -1,97 +1,126 @@
-import partiesDb from '../models/parties';
+import { Pool } from 'pg';
+
+const pool = new Pool({
```
Comment (tutugodfrey) [inline server/controllers/parties.js:3]: You have made use of this pool in more that one place now, and its use is likely to increase. How about abstracting into a function.

Comment (ibkadeeko) [reply server/controllers/parties.js:3]: I first abstracted it based on the documentation on the node-postgres.com site and was calling it in the different pages but at some point, my code was not working properly and doing it this way was what made it to work

Comment (tutugodfrey) [reply server/controllers/parties.js:3]: Ok

**Pull request closed without merging**